pkg/local-storage/member/controller/manager.go

```diff
@@ -243,6 +244,20 @@ func (m *manager) handleK8sNodeUpdatedEvent(oldObj, newObj interface{}) {
 	}
 }
 
+func (m *manager) handleVolumeCRDUpdateEvent(oldObj, newObj interface{}) {
+	oldVol := oldObj.(*apisv1alpha1.LocalVolume)
+	newVol := newObj.(*apisv1alpha1.LocalVolume)
+
+	// if volume's replica update, we should notify its group
+	if !reflect.DeepEqual(oldVol.Spec.Accessibility.Nodes, newVol.Spec.Accessibility.Nodes) {
```

Comment:
what will happen if nodes in accessibility array are same but in a different order？